schema_salad/tests/util.py

```diff
@@ -13,5 +19,25 @@ def get_data(filename):  # type: (Text) -> Optional[Text]
     except ResolutionError:
         pass
     if not filepath or not os.path.isfile(filepath):
-        filepath = os.path.join(os.path.dirname(__file__), os.pardir, filename)
+        # First try to load it from the local directory, probably ``./tests/``.
+        filepath = os.path.join(os.path.dirname(__file__), filename)
+        if not os.path.isfile(filepath):
+            # If that didn't work, then default to tests/../${filename},
+            # note that we return the parent as it is expected that __file__
+            # is a test file.
+            filepath = os.path.join(os.path.dirname(__file__), os.pardir, filename)
     return filepath
+
+
+def get_data_uri(resource_path: str) -> str:
+    """Get the file URI for tests."""
+    path = get_data(resource_path)
+    assert path
+    return ref_resolver.file_uri(path)
+
+
+# Schemas used in tests
+
+cwl_file_uri = get_data_uri("tests/test_schema/CommonWorkflowLanguage.yml")
+metaschema_file_uri = get_data_uri("metaschema/metaschema.yml")
+basket_file_uri = get_data_uri("basket.yml")
```

Comment:
The schema used by OP when reporting the issue. Note that we have another Basket schema being added in #542 that could be used for this test too, since it doesn't have `$base`.